airmozilla/main/models.py

```diff
@@ -213,6 +214,11 @@ def get_queryset(self):
     def scheduled(self):
         return self.get_query_set().filter(status=Event.STATUS_SCHEDULED)
 
+    def scheduled_or_processing(self):
+        return self.get_query_set().filter(Q(status=Event.STATUS_SCHEDULED)
+                                           |
+                                           Q(status=Event.STATUS_PROCESSING))
+
```

Comment:
I know it's petty but it actually matters. Can you reformat the indentation so it's like this instead:
```python
return self.get_query_set().filter(
    Q(status=Event.STATUS_SCHEDULED)
    |
    Q(status=Event.STATUS_PROCESSING)
)
```

The reason why it matters is because consistency is important. What's also important is to keep patches as small as possible. For example, if you wrote this:
```python
names = ('Peter',
         'Gloria',
         'Lisa',
         'Erica',
         'Richard',
)
```
and some day you decide to change the variable from `names` to `first_names` for some good reason. Then it would end up misaligned like this:
```python
first_names = ('Peter',
         'Gloria',
         'Lisa',
         'Erica',
         'Richard',
)
```
You're only intending on changing the variable name. Not the list/tuple. The list of names now look odd you think so you decide to change it. All of a sudden your diff becomes that of 5 lines. Perhaps that could cause a git conflict with someone elses changes even though all you changed was indentation. Also a code reviewing might see that change and have to study the difference very careful and think to herself "Hmm... Why did all of these names change? Let me look through each one."
A much better approach is this:
```python
names = (
    'Peter',
    'Gloria',
    'Lisa',
    'Erica',
    'Richard',
)
``` 
Now, the length of the variable `names` is independent of the list of names. The code is thus more "sustainable".